src/components/Cards/CardLinkWithIcon.js

```diff
@@ -0,0 +1,26 @@
+import ClassNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CardLinkWithIcon = ({ href, children, className, ...props }) => {
```

Comment:
my preference would be to rename it `CardLink` instead and use the same component for rendering "Footer link" (link w/o icon) and "Add New Cluster" (link w/ icon as children).